zarr_creator/config.py

```diff
@@ -1,84 +1,90 @@
 # The "data collection" may contain multiple named parts (each will be put in its own zarr archive)
 # Each part may contain multiple "level types" (e.g. heightAboveGround, etc)
 # and a name-mapping may also be defined
+from collections import OrderedDict
 
-DATA_COLLECTION = dict(
-    parts=dict(
-        height_levels=dict(
-            heightAboveGround=dict(
-                variables={
-                    # v: [30, 50, 75, 100, 150, 200, 250, 300, 500]  # levels in DANRA
-                    v: [50, 100, 150, 250]  # only these in DINI
-                    for v in "t r u v".split()
-                }
-            )
+DATA_COLLECTION = OrderedDict(
+    single_levels=[
+        dict(
+            level_type="heightAboveGround",
+            variables={
+                v: None
+                for v in [
+                    "hcc",
+                    "lcc",
+                    "mcc",
+                    # "tcc", # not in DINI
```

Comment:
We do have "tcc" in dini. It is just called `cc` as shortName.
Defined at 0m heightAboveGround.

Reply:
Ok, I will make a note to add this. This will not match with DANRA in that case. And so we should adapt a cf-standard name